src/app/cancel-mandate/cancel-mandate.component.html

```diff
@@ -0,0 +1,57 @@
+<div id="banner"></div>
+<main>
+  <div>
+    <biggive-page-section>
+      <div class="heading-wrapper">
+        <biggive-heading
+          colour="tertiary"
+          size="1"
+          align="center"
+          [text]="'Cancel regular giving to ' + mandate.charityName"
+        ></biggive-heading>
+      </div>
+      <!-- @todo  DON-1022 replace form with message if mandate is already cancelled has a non-cancelable status    -->
+      <p>
+        You are currently giving <strong>{{ mandate.donationAmount | money }}</strong> to
+        <strong>{{ mandate.charityName }}</strong>,
+        on the {{ mandate.schedule.dayOfMonth | ordinal }} of each month.
+        Click below to stop future donations.
+      </p>
+
+      <p>This will not affect any donations that have already been collected or are currently being processed.</p>
+
+      <form [formGroup]="cancellationForm">
+        <biggive-text-input spaceBelow="4">
+          <label slot="label" for="reason">Please tell us why you wish to stop giving
+            to {{ mandate.charityName }}</label>
+          <input
+            slot="input"
+            matInput type="email" id="reason" formControlName="reason" autocapitalize="off"
+            [maxlength]="reasonMaxLength">
+        </biggive-text-input>
+
+        <div class="actions">
+          <div>
+            <a href="/my-account/regular-giving/{{mandate.id}}">Back to mandate details</a>
+          </div>
+          <div>
+            @if (processing) {
+              <mat-spinner color="primary" diameter="30" aria-label="Processing"></mat-spinner>
+            } @else {
```

Comment:
the spinner seems to never be appearing in my tests, not sure why. Doesn't matter if things work quickly but we should give some indication that something is happening if the connection or matchbot is slow.

Reply:
@bdsl Your component might need `MatProgressSpinnerModule`?

Reply:
Thanks I'll check that tomorrow.